<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="1.394" timestamp="2025-01-09T10:25:36.279635" hostname="dorje"><testcase classname="" name="tests.model.test_model" time="0.000"><error message="collection failure">tests/model/test_model.py:6: in &lt;module&gt;
    from challenge.model import DelayModel
challenge/model.py:5: in &lt;module&gt;
    class DelayModel:
challenge/model.py:16: in DelayModel
    ) -&gt; Union(Tuple[pd.DataFrame, pd.DataFrame], pd.DataFrame):
../../../../.pyenv/versions/3.9.21/lib/python3.9/typing.py:344: in __call__
    raise TypeError(f"Cannot instantiate {self!r}")
E   TypeError: Cannot instantiate typing.Union</error></testcase></testsuite></testsuites>